Cloud_Community_REST_API_Docs.md

```
@@ -0,0 +1,1060 @@
+
+<aside>
+<img src="notion://custom_emoji/845a6cfa-ad4b-4505-8350-960c9f51a87a/168954da-c755-8023-8dcf-007afaa4b2e6" alt="notion://custom_emoji/845a6cfa-ad4b-4505-8350-960c9f51a87a/168954da-c755-8023-8dcf-007afaa4b2e6" width="40px" />
+
+전체화면으로 해놓고 구현하시면 편합니다!
+Cmd + T (Ctrl + T) 누르면 탭 추가가 가능합니다. 참고하세요!
+
+창 추가하는건 Cmd + Shift + N (Ctrl + Shift + N) 입니다!.
+
+</aside>
+
+### 내가 좋아요한 게시물 목록
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)